content/applications/inventory_and_mrp/inventory/routes/strategies/putaway.rst

```diff
@@ -1,54 +1,209 @@
+To manage where specific products are routed for storage, navigate to :menuselection:`Inventory -->
+Configuration --> Putaway Rules`. Use the :guilabel:`Create` button to configure a new putaway rule
+on a :guilabel:`Product` and/or :guilabel:`Product Category` that the rule affects.
 
-In some cases, like for a retail shop storing vegetables and fruits, we have to store products in
-different locations to maintain product quality.
+In the same line, the :guilabel:`When product arrives in` location is where the putaway rule is
+triggered to create an operation to move the product to the :guilabel:`Store to` location. The
+:guilabel:`Store to` location must be sublocation of the first (ie. `WH/Stock/Fruits` is a specific,
+named location inside `WH/Stock` to make the products stored here easier to find).
 
-Let’s suppose there are one warehouse location *WH/Stock* and two sub-locations
-*WH/Stock/Vegetables* and *WH/Stock/Fruits*.
+.. example::
+   Ensure all apples are stored in the fruits section by filling field :guilabel:`Store to` with the
+   location `WH/Stock/Fruits` when the :guilabel:`Product`, `Apple` arrives in `WH/Stock`. Repeat
+   this for all products and hit :guilabel:`Save`.
 
-To manage those locations, we will create putaway rules. To do so, open the *Inventory* app and go
-to :menuselection:`Configuration --> Putaway Rules`. Then, click on create and configure your first
-rule indicating the main location the product will enter before being redirected to the right
-location.
+   .. image:: putaway/create-putaway-rules.png
+      :align: center
+      :alt: Create putaway rules for apples and carrots.
 
 .. note::
-   The putaway rules can be defined either per product or per product category.
+  It is also possible to create and manage putaway rules for a single product by going to the
+  product page and clicking the :guilabel:`Putaway Rules` smart button at the top of the page. If
+  the button isn't there at first glance, select the :guilabel:`More` button at the top right to
+  view additional configuration options.
+
+.. _inventory/putaway/internal-stock-moves:
+
+.. tip::
+   Check whether putaway rules work as intended and using the summary of internal product movements.
+   In the :guilabel:`Inventory` app, select :menuselection:`Reporting --> Product Moves` and enable
+   the :guilabel:`Internal` search filter on the :guilabel:`Filters` dropdown under the search bar
+   at the top of the page. In this view, confirm that products are moved to the :guilabel:`Store to`
+   location detailed in the putaway rule. More details in :ref:`this section
+   <inventory/putaway/wrong-location>`.
+
+Use putaway rules
+-----------------
+
+Once a putaway rule has been configured, the product it specifies is automatically routed to the
+:guilabel:`Store to` location upon arriving in the :guilabel:`When product arrives in` location.
+
+After configuring the putaway rules for apples and carrots, see the internal stock moves by
```

Comment:
I would delete "apples and carrots" here to keep it more general rather than hypothetical.
```suggestion
After configuring the putaway rules, see the internal stock moves by
```